hosted/aks/p1/p1_provisioning_test.go

```diff
@@ -224,6 +260,29 @@ var _ = Describe("P1Provisioning", func() {
 
 	})
 
+	It("should successfully create 2 clusters in the same RG", func() {
+		testCaseID = 217
+		rgName := namegen.AppendRandomString("custom-aks-rg")
+		updateFunc := func(aksConfig *aks.ClusterConfig) {
+			aksConfig.ResourceGroup = rgName
+		}
+		var wg sync.WaitGroup
+		for i := 1; i <= 2; i++ {
+			wg.Add(1)
+			go func() {
+				defer wg.Done()
+				clusterName := namegen.AppendRandomString(helpers.ClusterNamePrefix)
+				cluster1, err := helper.CreateAKSHostedCluster(ctx.RancherAdminClient, clusterName, ctx.CloudCredID, k8sVersion, location, updateFunc)
```

Comment:
```suggestion
				cluster1, err := helper.CreateAKSHostedCluster(ctx.RancherAdminClient, clusterName, ctx.CloudCredID, k8sVersion, location, updateFunc)
```
wherever applicable

Reply:
Good catch. But I changed it to cluster1 instead so that it shows separate context and not as a part of the cluster created in every BeforeEach.